### DISCUSSION TITLE
IEEE 802.11 Fragmentation - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#618] [Showcases]
Discuss the "IEEE 802.11 Fragmentation" showcase here. Ideas and recommendations are welcome.

**Comment by gokseninarikan** [comment]
I think link is broken for this project. I can not reach ParameterStudy.anf file.

**Comment by adamgeorge309** [comment]
There is no link for ParameterStudy.anf, it's just mentioned. The file is in the fragmentation showcase's folder.

To be more precise, the file is not in any INET release yet, but it's available in the INET master branch.
